libs/components/src/search/vl-search.uig-css.ts

```diff
@@ -112,7 +112,7 @@ const styles: CSSResult = css`
     .vl-search--inline slot[name='input']:focus,
     .vl-search--inline .vl-search__input:focus + .vl-search__submit,
     .vl-search--inline slot[name='input']:focus + .vl-search__submit {
-        transition: opacity 0.2s, transform 0.2s;
+        // transition: opacity 0.2s, transform 0.2s;
```

Comment:
deze lijn gewoon verwijderen, niet in commentaar zetten

Reply:
Done.